caf-audit-maven-plugin/src/main/java/com/hpe/caf/auditing/plugins/XMLToJavaMojo.java

```diff
@@ -50,24 +51,43 @@
     @Parameter(property = "packageName", defaultValue = "${project.groupId}.auditing")
     private String packageName;
 
+    /**
+     *  Indicates if the generated Java should be added as a test resource. If not set, or set to false, it is added as
+     *  a main resource.
+     */
+    @Parameter(property = "generateAsTestResource", defaultValue = "false")
```

Comment:
Check with Dermot if this new parameter needs documented under audit plugin sections of caf-audit README, Getting Started and Client-Side API pages.